api4/user.go

```diff
@@ -942,8 +942,29 @@ func login(c *Context, w http.ResponseWriter, r *http.Request) {
 	deviceId := props["device_id"]
 	ldapOnly := props["ldap_only"] == "true"
 
+	certPem, certSubject, certEmail := c.App.CheckForClienSideCert(r)
+	mlog.Info(fmt.Sprintf("Client Cert PEM: %v", certPem))
```

Comment:
We probably want this on debug. Also avoid the Sprintf by using a field. (you can combine these three lines into one)